services/api-server/src/simcore_service_api_server/services/log_streaming.py

```diff
@@ -53,34 +53,27 @@ async def __aexit__(self, exc_type, exc, tb):
         await self.teardown()
 
     async def _distribute_logs(self, data: bytes):
+        queue: Queue | None = None
         try:
-            got = LoggerRabbitMessage.parse_raw(
-                data
-            )  # rabbitmq client safe_nacks the message if this deserialization fails
-        except ValidationError as e:
-            _logger.debug(
-                "Could not parse log message from RabbitMQ in LogDistributor._distribute_logs"
+            got = LoggerRabbitMessage.parse_raw(data)
+            item = JobLog(
+                job_id=got.project_id,
+                node_id=got.node_id,
+                log_level=got.log_level,
+                messages=got.messages,
             )
-            raise e
-        _logger.debug(
-            "LogDistributor._distribute_logs received message message from RabbitMQ: %s",
-            got.json(),
-        )
-        item = JobLog(
-            job_id=got.project_id,
-            node_id=got.node_id,
-            log_level=got.log_level,
-            messages=got.messages,
-        )
-        queue = self._log_streamers.get(item.job_id)
-        if queue is None:
-            raise LogStreamerNotRegistered(
-                f"Could not forward log because a logstreamer associated with job_id={item.job_id} was not registered"
-            )
-        await queue.put(item)
+            queue = self._log_streamers.get(item.job_id)
+            if queue is None:
+                raise LogStreamerNotRegistered(
+                    f"Could not forward log because a logstreamer associated with job_id={item.job_id} was not registered"
+                )
+            await queue.put(item)
+        except Exception as exc:  # pylint: disable=broad-except
```

Comment:
what about:
```python
with log_catch()...:
    got = LoggerRabbitMessage.parse_raw(data)
    item = JobLog(
        job_id=got.project_id,
        node_id=got.node_id,
        log_level=got.log_level,
        messages=got.messages,
    )
    queue = self._log_streamers.get(item.job_id)
    if queue is None:
        raise LogStreamerNotRegistered(
            f"Could not forward log because a logstreamer associated with job_id={item.job_id} was not registered"
        )
    await queue.put(item)
    return True
return False
```